juriscraper/OpinionSiteLinear.py

```diff
@@ -70,3 +70,6 @@ def _get_parallel_citations(self):
 
     def _get_summaries(self):
         return self._get_optional_field_by_id("summary")
+
+    def _get_west_citations(self):
+        return self._get_optional_field_by_id("west_citation")
```

Comment:
Isn't this no-longer a thing?

Reply:
ugh ... no - if it is my rebase was just faulty.

Reply:
ah I see this was just missed in the previous release and not a failed rebase.

Reply:
actually I have no idea how that got there.

Reply:
dropped this random addition